languages.toml

```
@@ -423,6 +423,18 @@ indent = { tab-width = 2, unit = "  " }
 name = "html"
 source = { git = "https://github.com/tree-sitter/tree-sitter-html", rev = "d93af487cc75120c89257195e6be46c999c6ba18" }
 
+[[language]]
+name = "htmldjango"
+scope = "source.htmldjango"
+injection-regex = "htmldjango"
+file-types = ["html"]
```

It looks like this entry beats regular html for `.html` files. We have another conflict like this between verilog and V and I think the only way around it for now is to have one of them where you have to override it in a `.helix/languages.toml` or `~/.config/helix/languages.toml`:

```toml
[[language]]
name = "html"
file-types = []

[[language]]
name = "htmldjango"
file-types = ["html"]
```

Since HTML is more popular than django I think we should disable this language by default

```suggestion
file-types = []
```

---

Yes, it beats `html`, but doesn't it only apply if say the root file `manage.py` is found?

---

Oh wait, it doesn't. Maybe we should apply it conditionally based on the `roots`?

---

I was also under the impression that `file-types` would only be superceded if `manage.py` was found based on these settings, a file I've never heard of or seen people using until I did some web searching based on this PR's options. I'll need to re-read those options.

---

This should be fixed now, but I am still a bit not quite happy with the result that the ordering is a bit messy (it will take the first glob that matches the path and have the root within the path). Some parts of it was taken from https://github.com/helix-editor/helix/pull/2455, but I haven't updated the docs yet.

---

This should be fixed now, but I am still a bit not quite happy with the result that the ordering is a bit messy (it will take the first glob that matches the path and have the root within the path). Some parts of it was taken from https://github.com/helix-editor/helix/pull/2455, but I haven't updated the docs yet.